doc: Add guide for Daizy cloud integration (#865)
TheThingsIndustries/lorawan-stack-docs: doc/content/integrations/cloud-integrations/daizy/index.md

---
title: "Daizy"
description: ""
weight:
---

Use [Daizy](https://daizy.io) to rapidly build, deploy and scale Internet of Things applications and stay in control of your data.

The Daizy platform provides an intuitive management system for IoT devices and projects, and delivers data directly to any existing or future business systems. Daizy enables rapid scaling by reducing time to market for any IoT implementation while minimizing cost, improving flexibility and maintaining security.

<!--more-->

## Prerequisites

1. A user account and organisation created within the [Daizy platform](https://portal.daizy.io/login).

> 2. LoRa devices enrolled and deployed in your project.

Reviewer: This is redundant, we assume devices are already sending data to TTS for all integrations

Author: The intent here was that the user needs the devices enrolled in their Daizy project, not in TTS, I will make this clear.

## Set up Daizy

> The details required for creating the webhook can be found under Compose > Connectivity > IoT Endpoint

Reviewer: ```suggestion
The details required for creating the webhook on Daizy can be found under **Compose &#8594; Connectivity &#8594; IoT Endpoint**.
```

{{< figure src="connectivity.png" alt="Daizy Connectivity tab" >}}

> Copy the values for the Organisation Slug, and Uplink API Key.

Reviewer: ```suggestion
Copy the **Organisation slug** and **Uplink API key** values.
```

## Configure {{% tts %}}

In {{% tts %}}, create a webhook using the ready made **Daizy** [Webhook template]({{< ref "/integrations/webhooks/webhook-templates" >}}).

Give your Webhook a name in the **Webhook ID** field.

> Copy the **Organisation slug** and **Uplink API Key** from your **IoT Endpoint** details screen in Daizy, and fill these in accordingly.

Reviewer: ```suggestion
Paste the **Organisation slug** and **Uplink API key** values you copied in the previous steps to the corresponding fields of the Webhook template.
```
